lib/config/folderconfiguration.go

```diff
@@ -84,6 +84,11 @@ func (f *FolderConfiguration) CreateMarker() error {
 			return err
 		}
 		fd.Close()
+		if os.Getenv("STFSYNC") != "" {
+			if syncErr := osutil.SyncDir(filepath.Dir(marker)); syncErr != nil {
```

Comment:
The `syncErr`s should typically all be called `err`, otherwise I'm all confused wondering why it's different and if there's an `err` declared somewhere we must not shadow etc.